modules/persistent-storage-csi-gcp-file-install.adoc

```diff
@@ -6,16 +6,30 @@
 [id="persistent-storage-csi-olm-operator-install_{context}"]
 = Installing the GCP Filestore CSI Driver Operator
 
-The Google Compute Platform (GCP) Filestore Container Storage Interface (CSI) Driver Operator is not installed in {product-title} by default. Use the following procedure to install the GCP Filestore CSI Driver Operator in your cluster.
+The Google Compute Platform (GCP) Filestore Container Storage Interface (CSI) Driver Operator is not installed in {product-title} by default.
+Use the following procedure to install the GCP Filestore CSI Driver Operator in your cluster.
 
 .Prerequisites
 * Access to the {product-title} web console.
 
 .Procedure
 To install the GCP Filestore CSI Driver Operator from the web console:
 
+ifdef::openshift-dedicated[]
+
+. Log in to the {cluster-manager-url} and click the name of your cluster.
+
+. Select your cluster.
```

Comment:
The previous step says to click the name of your cluster, and this step says to select your cluster. Is there a difference between these two actions? 

If this comment makes no sense when you're actually looking at the console, feel free to disregard!

Reply:
Hi @bhardesty,
   Makes total sense and thanks for catching. Seemed to make sense on Friday, not so much today! :) 
Have updated and removed the repeated step.